app/javascript/article-form/articleForm.jsx

```diff
@@ -211,9 +206,7 @@ export default class ArticleForm extends Component {
     this.setState({ organizationId });
   };
 
-  failedPreview = response => {
-    console.log(response);
-  };
```

Comment:
albeit not great, I assume this was on purpose, otherwise this error callback would be pointless. Maybe we can tell eslint to ignore the `console.log` line for now and come up with a better solution in a separate PR?

Reply:
That is fine with me. 👍

Reply:
shall I put a FIXME or a TODO comment?

Reply:
AFAIK there are no `FIXME` comments in the code, so let's use a `TODO` :-)

thanks!

Reply:
Indeed. There's no FIXME, just TODO :)